src/Css/Style.php

```diff
@@ -1581,7 +1586,7 @@ protected function clear_cache(string $prop): void
     public function set_prop(string $prop, $val, bool $important = false, bool $clear_dependencies = true): void
     {
         // Skip some checks for CSS custom properties.
-        if (substr($prop, 0, 2) !== "--") {
+        if (!(\strpos($prop, '--') === 0)) {
```

Comment:
Should we call is_custom_property for this one as well?

Reply:
Ah, not sure how I've overlooked that one. Will be there in a minute.